main.py

```diff
@@ -0,0 +1,114 @@
+import fitz
+import openai
+from nltk.tokenize import sent_tokenize
+from io import StringIO
+import json
+import os
+import sys
+
+def open_file(filepath):
+  with open(filepath, 'r', encoding='utf-8') as infile:
+    return infile.read()
+
+my_secret = os.environ['OPENAI_API_KEY']
```

Comment:
/wtd use a try statement to catch if the API key is not define, and show a warning if needed